operations/helm/charts/alloy/values.yaml

```diff
@@ -41,6 +41,9 @@ alloy:
     # -- Deploy Alloy in a cluster to allow for load distribution.
     enabled: false
 
+    # -- Name for the port used for clustering, useful if running inside an Istio Mesh
+    portName: null
```

Comment:
I generally prefer the values.yaml exposing as much information as possible about what default values are; right now you'd have to dig into the templates to see that if `portName` is null, then the port name is `http`. 

Instead of using null/coalesce for the portName, can we set this to `portName: http` instead? That way the values.yaml are a little more self-documenting.

Reply:
done